tests/org.eclipse.n4js.packagejson.xpect.ui.tests/xpect/GH_991_nested-node-modules-folder/P/package.json.xt

```diff
@@ -23,6 +23,10 @@
         }
     }
 
+
+	IssueConfiguration {
+		IssueCode "MISSING_YARN_WORKSPACE" { enabled = false }
```

Comment:
have you considered using `N4JSLanguageConstants#DEFAULT_SUPPRESSED_ISSUE_CODES_FOR_TESTS`?

Reply:
Note: Both of us discussed this afterwards and chose to leave it like that. Reason was that it should be decided on the test case whether the issue needs to be shown or hidden.